klippy/mcu.py

```diff
@@ -682,7 +688,12 @@ def _shutdown(self, force=False):
     def _restart_arduino(self):
         logging.info("Attempting MCU '%s' reset", self._name)
         self._disconnect()
-        serialhdl.arduino_reset(self._serialport, self._reactor)
+        serialhdl.arduino_reset(self._serialport, self._reactor,
+                                self._serial_rts)
```

Comment:
This looks like an error to me, as arduino_reset() no longer takes a third parameter.

Also, fwiw, I think serial_rts could just be a local variable in the `__init__()` method now.

-Kevin

Reply:
Both fixed.